shared/desktop/app/index.js

```diff
@@ -26,6 +27,10 @@ function start() {
       if (mainWindow) {
         mainWindow.show(true)
       }
+      if (isWindows) {
+        // Wasn't showing on Windows without this
+        showMainWindow()
```

Comment:
Not sure, but it doesn't work without it in Windows. Isn't the current app the second one in this case, which may not yet have received 'ready', the handler for which populates `mainWindow`?